dags/databricks-automl-example.py

```diff
@@ -0,0 +1,64 @@
+from datetime import datetime
+import logging
+
+from airflow.decorators import task, dag
+from airflow.providers.databricks.hooks.databricks import DatabricksHook
+from airflow.providers.databricks.operators.databricks import DatabricksSubmitRunOperator
+
+import mlflow
+
+docs = """
+Demonstrates orchestrating ML pipelines executed on Databricks with Airflow
+"""
+
+
+@dag(
+    start_date=datetime(2022, 1, 1),
+    schedule_interval=None,
+    catchup=False,
+    doc_md=docs
+)
+def databricks_automl_example():
+
+    ingest_notebook = {
+        'notebook_path': "/Users/{{ var.value.databricks_user }}/BigQuery_to_Databricks",
+    }
+
+    ingest = DatabricksSubmitRunOperator(
+        task_id='ingest_notebook_task',
+        existing_cluster_id="{{ var.value.databricks_cluster_id }}",
+        notebook_task=ingest_notebook
```

Comment:
A stylistic comment but IMO it's a little more readable to put the value for `notebook_task` right in the operator instantiation so your eye doesn't have to look back up at the `ingest_notebook` value to reference it. Same comment for the other `notebook_task` args. No strong opinion though.

Reply:
Don't disagree with this, but thinking the other dag should also be formatted similarly then